flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/stream/PartitionTimeCommitPredicate.java

```diff
@@ -56,6 +57,7 @@ public PartitionTimeCommitPredicate(
                         cl,
                         conf.get(PARTITION_TIME_EXTRACTOR_KIND),
                         conf.get(PARTITION_TIME_EXTRACTOR_CLASS),
+                        conf.get(PARTITION_TIME_EXTRACTOR_TIMESTAMP_FORMATTER),
                         conf.get(PARTITION_TIME_EXTRACTOR_TIMESTAMP_PATTERN));
```

Comment:
minor： move argument TIMESTAMP_FORMATTER to be after TIMESTAMP_PATTERN for readable consideration?

Reply:
done. thx.